packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -59,9 +77,11 @@ contract LongShortPair is Testable, Lockable {
 
     LongShortPairFinancialProductLibrary public financialProductLibrary;
 
+    // Optimistic oracle customization parameters.
     bytes public customAncillaryData;
-
     uint256 public prepaidProposerReward;
+    uint256 public optimisticOracleLivenessTime;
```

Comment:
nit: not sure how much stack space we need to save, but if we only need to save like one variable, we could create a struct called OptimisticOracleParams that just contains the reward, liveness, and bond. That might get around the stack limit without creating so many changes. We don't have to do this, as it may be less clean, but it could reduce the lines changed so this is easier to get audited.